app/assets/javascripts/orders.js

```
@@ -0,0 +1,311 @@
+		success: function(returnData){
+			// reload order items table
+		},
+		error: function(status, jqXHR, returnText){
+			console.log(status);
+		}
+		
+	});
+}
+
+var delId = "";
+function changeDeleteId(bikeRow){
```

Biome (1.9.4)

[error] Shouldn't redeclare 'changeDeleteId'. Consider to delete it or rename it.

'changeDeleteId' is defined here:


(lint/suspicious/noRedeclare)